static/src/stylesheets/module/commercial/glabs/_hosted-article.scss

```diff
@@ -126,60 +144,172 @@ $zootropolisColor: #2ec869; //need to remove references to brand colours here ev
         }
 
         @include mq(desktop) {
-            left: ($gs-column-width * 4 + 20) * -1;
+            left: ($gs-gutter * 3) * -1;
             top: 20px;
+            width: gs-span(13);
+        }
+
+        @include mq(leftCol) {
+            left: ($gs-column-width * 4 + 20) * -1;
             width: gs-span(16) + $gs-gutter * 2;
         }
     }
+    @mixin animal-stripe-character {
+        z-index: 2;
+        position: absolute;
+        padding-bottom: 0 !important; //u-responsive-ratio has inline styles which sets padding-bottom to 100% to keep the proper ratio but I dont need to care about this here
+    }
     .animal {
         position: relative;
-        margin-top: 40px;
+        margin-top: 30px;
 
         &.con-artist {
-            .animal__stripe-character {
-                position: absolute;
-                top: -117px;
-                left: 42px;
+            .u-responsive-ratio {
```

Comment:
I know, the whole thing with hosting those animals via grid is a mess and I am not sure if this is the best method. I could host them simply on AWS but needed to crop them first.

Anyway, this is the only example of article with such custom elements. We are going to build something generic (without those animal stripes) so we will get rid of that mess.